tests/Behat/Page/Admin/Product/UpdateSimpleProductPage.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\BitBag\SyliusProductAttributeGroupsPlugin\Behat\Page\Admin\Product;
+
+use Sylius\Behat\Page\Admin\Crud\UpdatePage;
+
+class UpdateSimpleProductPage extends UpdatePage
+{
+    public function openTab(string $tabName): void
+    {
+        $attributesTab = $this->getElement('tab', ['%name%' => $tabName]);
+        if (!$attributesTab->hasClass('active')) {
+            $attributesTab->click();
+        }
+
+        $this->getSession()->wait(1000);
```

Comment:
I think we can wait for the action to load instead of waiting a while
(what is in sylius behats which tests js's ?)

Reply:
wait removed

Reply:
changed to waitForElement like in sylius behats